packages/ui/VirtualList/VirtualList.module.less

```diff
@@ -18,6 +18,10 @@
 
 		will-change: transform;
 	}
+	
+	.content {
```

Comment:
* Renamed to `.content` to use semantic names rather than behavioral names
* removed the direct descendant selector (`>`) which wasn't necessary